api/FetchEvent.json

```
@@ -356,10 +356,10 @@
           "spec_url": "https://w3c.github.io/ServiceWorker/#fetch-event-request",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "40"
```

This change, but everywhere and mirrored:

```suggestion
              "version_added": "42"
```